nitrite test fails to compile to native with Camel 4 and Quarkus 3

```
2023-02-07T15:17:59.8446702Z GraalVM Native Image: Generating 'camel-quarkus-integration-test-nitrite-3.0.0-SNAPSHOT-runner' (executable)...
2023-02-07T15:17:59.8450238Z ========================================================================================================================
2023-02-07T15:18:04.2776120Z [1/7] Initializing...                                                                                   (12.5s @ 0.30GB)
2023-02-07T15:18:04.2789514Z  Version info: 'GraalVM 22.3.1.0-Final Java 17 Mandrel Distribution'
2023-02-07T15:18:04.2792559Z  Java version info: '17.0.6+10'
2023-02-07T15:18:04.2801042Z  C compiler: gcc (redhat, x86_64, 8.5.0)
2023-02-07T15:18:04.2804652Z  Garbage collector: Serial GC
2023-02-07T15:18:04.2821827Z  3 user-specific feature(s)
2023-02-07T15:18:04.2823254Z  - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
2023-02-07T15:18:04.2831125Z  - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
2023-02-07T15:18:04.2836163Z  - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
2023-02-07T15:19:03.7194694Z [2/7] Performing analysis...  []                                                                        (59.1s @ 1.62GB)
2023-02-07T15:19:03.7252417Z   11,340 (92.59%) of 12,248 classes reachable
2023-02-07T15:19:03.7329208Z   14,994 (58.87%) of 25,471 fields reachable
2023-02-07T15:19:03.7624962Z   45,711 (80.78%) of 56,588 methods reachable
2023-02-07T15:19:03.7625449Z    2,466 classes,     0 fields, and     0 methods registered for reflection
2023-02-07T15:19:03.7655219Z 
2023-02-07T15:19:03.7677802Z Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing uk.co.jemos.podam.api.PodamFactoryImpl.getParameterValuesForConstructor(java.lang.reflect.Constructor, java.lang.Class, uk.co.jemos.podam.common.ManufacturingContext, java.util.Map, java.lang.reflect.Type[]) 
2023-02-07T15:19:03.7680469Z Parsing context:
2023-02-07T15:19:03.7685180Z    at uk.co.jemos.podam.api.PodamFactoryImpl.getParameterValuesForConstructor(PodamFactoryImpl.java:1749)
2023-02-07T15:19:03.7688835Z    at uk.co.jemos.podam.api.PodamFactoryImpl.instantiatePojo(PodamFactoryImpl.java:382)
2023-02-07T15:19:03.7696205Z    at uk.co.jemos.podam.api.PodamFactoryImpl.manufacturePojoInternal(PodamFactoryImpl.java:505)
2023-02-07T15:19:03.7696964Z    at uk.co.jemos.podam.api.PodamFactoryImpl.doManufacturePojo(PodamFactoryImpl.java:428)
2023-02-07T15:19:03.7701200Z    at uk.co.jemos.podam.api.PodamFactoryImpl.manufacturePojo(PodamFactoryImpl.java:150)
2023-02-07T15:19:03.7704412Z    at org.dizitart.no2.util.DocumentUtils.dummyDocument(DocumentUtils.java:139)
2023-02-07T15:19:03.7706165Z    at org.dizitart.no2.util.ValidationUtils.validateObjectIndexField(ValidationUtils.java:231)
2023-02-07T15:19:03.7706670Z    at org.dizitart.no2.util.ObjectUtils.getIdField(ObjectUtils.java:148)
2023-02-07T15:19:03.7707285Z    at org.dizitart.no2.objects.DefaultObjectRepository.createIndexes(DefaultObjectRepository.java:315)
2023-02-07T15:19:03.7712424Z    at org.dizitart.no2.objects.DefaultObjectRepository.initRepository(DefaultObjectRepository.java:303)
2023-02-07T15:19:03.7713191Z    at org.dizitart.no2.objects.DefaultObjectRepository.<init>(DefaultObjectRepository.java:57)
2023-02-07T15:19:03.7713698Z    at org.dizitart.no2.objects.RepositoryFactory.open(RepositoryFactory.java:46)
2023-02-07T15:19:03.7717473Z    at org.dizitart.no2.Nitrite.getRepository(Nitrite.java:168)
2023-02-07T15:19:03.7720125Z    at org.apache.camel.component.nitrite.NitriteComponent.lambda$createEndpoint$1(NitriteComponent.java:67)
2023-02-07T15:19:03.7723311Z    at org.apache.camel.component.nitrite.NitriteComponent$$Lambda$1447/0x00000007c1a8b5c0.apply(Unknown Source)
2023-02-07T15:19:03.7726454Z    at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
2023-02-07T15:19:03.7728981Z    at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
2023-02-07T15:19:03.7732575Z    at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
2023-02-07T15:19:03.7733280Z    at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290)
2023-02-07T15:19:03.7733781Z    at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:754)
2023-02-07T15:19:03.7736891Z    at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
2023-02-07T15:19:03.7737335Z    at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:686)
2023-02-07T15:19:03.7737822Z    at java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:159)
2023-02-07T15:19:03.7738332Z    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:173)
2023-02-07T15:19:03.7738854Z    at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
2023-02-07T15:19:03.7739309Z    at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
2023-02-07T15:19:03.7742839Z    at java.lang.reflect.Executable.sharedToString(Executable.java:122)
2023-02-07T15:19:03.7743418Z    at java.lang.reflect.Method.toString(Method.java:412)
2023-02-07T15:19:03.7746276Z    at java.lang.StringConcatHelper.stringOf(StringConcatHelper.java:453)
2023-02-07T15:19:03.7748619Z    at java.lang.StringConcatHelper.simpleConcat(StringConcatHelper.java:408)
2023-02-07T15:19:03.7754807Z    at com.oracle.svm.core.reflect.target.Target_java_lang_reflect_Method.acquireMethodAccessor(Target_java_lang_reflect_Method.java:76)
2023-02-07T15:19:03.7755507Z    at java.lang.reflect.Method.invoke(Method.java:566)
2023-02-07T15:19:03.7755931Z    at com.oracle.svm.core.containers.Metrics.systemMetrics(Metrics.java:63)
2023-02-07T15:19:03.7758979Z    at com.oracle.svm.core.containers.Container.metrics(Container.java:44)
2023-02-07T15:19:03.7759291Z 
2023-02-07T15:19:03.7761360Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
2023-02-07T15:19:03.7766411Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
2023-02-07T15:19:03.7767508Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
2023-02-07T15:19:03.7773319Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
2023-02-07T15:19:03.7774184Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultSpecialInvokeTypeFlow.onObservedUpdate(DefaultSpecialInvokeTypeFlow.java:61)
2023-02-07T15:19:03.7778907Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:562)
2023-02-07T15:19:03.7779486Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:488)
2023-02-07T15:19:03.7782379Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
2023-02-07T15:19:03.7790620Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
2023-02-07T15:19:03.7796531Z 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
2023-02-07T15:19:03.7797047Z 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
2023-02-07T15:19:03.7802303Z 	at java.base/java.util.concurrent.ForkJoinPool.externalHelpQuiescePool(ForkJoinPool.java:2104)
2023-02-07T15:19:03.7805859Z 	at java.base/java.util.concurrent.ForkJoinPool.awaitQuiescence(ForkJoinPool.java:3321)
2023-02-07T15:19:03.7807974Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.complete(CompletionExecutor.java:243)
2023-02-07T15:19:03.7811496Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.doTypeflow(PointsToAnalysis.java:541)
2023-02-07T15:19:03.7816182Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.finish(PointsToAnalysis.java:529)
2023-02-07T15:19:03.7816965Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.AbstractAnalysisEngine.runAnalysis(AbstractAnalysisEngine.java:143)
2023-02-07T15:19:03.7817715Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:745)
2023-02-07T15:19:03.7818363Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:578)
2023-02-07T15:19:03.7823658Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:535)
2023-02-07T15:19:03.7826713Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
2023-02-07T15:19:03.7827466Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:580)
2023-02-07T15:19:03.7833000Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
2023-02-07T15:19:03.7838257Z Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: javax.validation.Constraint. This error is reported at image build time because class uk.co.jemos.podam.typeManufacturers.TypeManufacturerUtil is registered for linking at image build time by command line
2023-02-07T15:19:03.7842986Z 	at parsing uk.co.jemos.podam.typeManufacturers.TypeManufacturerUtil.findAttributeStrategy(TypeManufacturerUtil.java:90)
2023-02-07T15:19:03.7845750Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
2023-02-07T15:19:03.7846508Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
2023-02-07T15:19:03.7847267Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3393)
2023-02-07T15:19:03.7847966Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3345)
2023-02-07T15:19:03.7851715Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3190)
2023-02-07T15:19:03.7852387Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
2023-02-07T15:19:03.7852958Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
2023-02-07T15:19:03.7859284Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:97)
2023-02-07T15:19:03.7860051Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
2023-02-07T15:19:03.7862988Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
2023-02-07T15:19:03.7863750Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:446)
2023-02-07T15:19:03.7864310Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
2023-02-07T15:19:03.7868409Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
2023-02-07T15:19:03.7869026Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
2023-02-07T15:19:03.7872700Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
2023-02-07T15:19:03.7873577Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.lookupEncodedGraph(InlineBeforeAnalysis.java:180)
2023-02-07T15:19:03.7879098Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.doInline(PEGraphDecoder.java:1162)
2023-02-07T15:19:03.7879734Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.tryInline(PEGraphDecoder.java:1145)
2023-02-07T15:19:03.7883214Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.trySimplifyInvoke(PEGraphDecoder.java:1003)
2023-02-07T15:19:03.7883923Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvoke(PEGraphDecoder.java:957)
2023-02-07T15:19:03.7886604Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.processNextNode(GraphDecoder.java:817)
2023-02-07T15:19:03.7887343Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.processNextNode(InlineBeforeAnalysis.java:240)
2023-02-07T15:19:03.7888084Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.decode(GraphDecoder.java:548)
2023-02-07T15:19:03.7888852Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.decode(PEGraphDecoder.java:833)
2023-02-07T15:19:03.7892168Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.phases.InlineBeforeAnalysis.decodeGraph(InlineBeforeAnalysis.java:98)
2023-02-07T15:19:03.7892865Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:179)
2023-02-07T15:19:03.7896421Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:349)
2023-02-07T15:19:03.7897138Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
2023-02-07T15:19:03.7897626Z 	... 21 more
2023-02-07T15:19:03.7900315Z Caused by: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: javax.validation.Constraint. This error is reported at image build time because class uk.co.jemos.podam.typeManufacturers.TypeManufacturerUtil is registered for linking at image build time by command line
2023-02-07T15:19:03.7903436Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.reportUnresolvedElement(SharedGraphBuilderPhase.java:333)
2023-02-07T15:19:03.7907309Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedType(SharedGraphBuilderPhase.java:288)
2023-02-07T15:19:03.7912453Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedLoadConstant(SharedGraphBuilderPhase.java:259)
2023-02-07T15:19:03.7913228Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genLoadConstant(BytecodeParser.java:3945)
2023-02-07T15:19:03.7915846Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5124)
2023-02-07T15:19:03.7916523Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3385)
2023-02-07T15:19:03.7919082Z 	... 46 more
```

--- a/extensions/nitrite/runtime/src/main/java/org/apache/camel/quarkus/component/nitrite/graal/PodamSubstitutions.java
+++ b/extensions/nitrite/runtime/src/main/java/org/apache/camel/quarkus/component/nitrite/graal/PodamSubstitutions.java
@@ -1,0 +1,107 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.nitrite.graal;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import uk.co.jemos.podam.api.AttributeMetadata;
+import uk.co.jemos.podam.api.ClassAttribute;
+import uk.co.jemos.podam.api.DataProviderStrategy;
+import uk.co.jemos.podam.api.MapArguments;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+import uk.co.jemos.podam.common.ManufacturingContext;
+import uk.co.jemos.podam.typeManufacturers.ArrayTypeManufacturerImpl;
+
+final public class PodamSubstitutions {
+
+}
+
+@TargetClass(PodamFactoryImpl.class)
+final class PodamFactoryImplSubstitutions {
+
+    @Substitute
+    private Object[] getParameterValuesForConstructor(
+            Constructor<?> constructor, Class<?> pojoClass,
+            ManufacturingContext manufacturingCtx,
+            Map map,
+            Type... genericTypeArgs)
+            throws InstantiationException, IllegalAccessException,
+            InvocationTargetException, ClassNotFoundException {
+        return new Object[0];
+    }
+
+    @Substitute
+    private Object[] getParameterValuesForMethod(
+            Method method, Class<?> pojoClass,
+            ManufacturingContext manufacturingCtx, Map<String, Type> typeArgsMap,
+            Type... genericTypeArgs)
+            throws InstantiationException, IllegalAccessException,
+            InvocationTargetException, ClassNotFoundException {
+        return new Object[0];
+    }
+
+    @Substitute
+    private void fillMap(MapArguments mapArguments, ManufacturingContext manufacturingCtx)
+            throws InstantiationException, IllegalAccessException,
+            InvocationTargetException, ClassNotFoundException {
+    }
+
+    @Substitute
+    private void fillCollection(ManufacturingContext manufacturingCtx,
+            List<Annotation> annotations, String attributeName,
+            Collection<? super Object> collection,
+            Class<?> collectionElementType, Type... genericTypeArgs)
+            throws InstantiationException, IllegalAccessException,
+            InvocationTargetException, ClassNotFoundException {
+    }
+
+    @Substitute
+    private void fillArray(Object array, String attributeName, Class<?> elementType,
+            Type genericElementType, List<Annotation> annotations,
+            ManufacturingContext manufacturingCtx,
+            Map<String, Type> typeArgsMap)
+            throws InstantiationException, IllegalAccessException,
+            InvocationTargetException, ClassNotFoundException {
+    }
+
+    @Substitute
+    private <T> boolean populateReadWriteField(T pojo, ClassAttribute attribute,
+            Map<String, Type> typeArgsMap, ManufacturingContext manufacturingCtx)
+            throws InstantiationException, IllegalAccessException,
+            InvocationTargetException, ClassNotFoundException {
+        return false;
+    }
+}
+
+@TargetClass(ArrayTypeManufacturerImpl.class)
+final class ArrayTypeManufacturerImplSubstitutions {
+    @Substitute
+    public Cloneable getType(DataProviderStrategy strategy,
+            AttributeMetadata attributeMetadata,
+            Map<String, Type> genericTypesArgumentsMap) {
+        return null;
+    }
+}

--- a/integration-tests/pom.xml
+++ b/integration-tests/pom.xml
@@ -166,7 +166,7 @@
         <module>mybatis</module>
         <module>nats</module>
         <module>netty</module>
-        <!--<module>nitrite</module> https://github.com/apache/camel-quarkus/issues/4525 -->
+        <module>nitrite</module>
         <module>oaipmh</module>
         <module>ognl</module>
         <module>olingo4</module>

--- a/tooling/scripts/test-categories.yaml
+++ b/tooling/scripts/test-categories.yaml
@@ -185,4 +185,5 @@ group-13:
   - sap-netweaver
   - servicenow
   - slack
+  - nitrite
   - pgevent

